src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
```

Comment:
```suggestion
    <App />
```
strict mode renders the page twice in dev mode, which was initing ulabel twice for me and breaking everything

Reply:
I also got double outputs from the initial `console.log` calls, but after load everything worked fine.
Obviously initializing twice is also a big problem, so going to commit this change. Good catch!